lib/hooks/peer-list-hooks.js

```diff
@@ -14,23 +15,21 @@ import { usePeerToPeerCommunication } from '../tunnelbroker/peer-to-peer-context
 import { useTunnelbroker } from '../tunnelbroker/tunnelbroker-context.js';
 import { useResendPeerToPeerMessages } from '../tunnelbroker/use-resend-peer-to-peer-messages.js';
 import type {
-  UsersRawDeviceLists,
-  UsersDevicesPlatformDetails,
-  SignedDeviceList,
   RawDeviceList,
+  SignedDeviceList,
```

Comment:
Concerning to see these moved around, makes me wonder if the prior PR would fail ESLint CI